app/models.py

```diff
@@ -724,7 +724,7 @@ def without_id(dictionary):
         if self.addresses:
             j['address'] = self.serialize_address(self.addresses[0])
 
```

Comment:
I spose it could if `self.addresses = []` then `self.address[0]` would fail - not sure that is the case ever though